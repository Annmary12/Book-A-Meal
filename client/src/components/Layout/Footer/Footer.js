import React from 'react';
import styles from './style.scss';

const Footer = () => (
  <footer className={styles.footer}>
    &copy; {new Date().getFullYear()} Book-A-Meal.
  </footer>
);

export default Footer;
