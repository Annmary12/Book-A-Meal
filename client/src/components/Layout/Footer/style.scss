.footer{
  text-align: center;
  background-color: #2e2d2d;
  color: white;
  padding: 10px;
  margin-top: 30px;
  flex-shrink: 0;
}