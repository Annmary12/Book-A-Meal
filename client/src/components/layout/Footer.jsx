import React from 'react';

const Footer = () => <footer>&copy; 2018 Book-A-Meal Inc.</footer>;

export default Footer;