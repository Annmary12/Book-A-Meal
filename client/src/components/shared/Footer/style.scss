.footer{
  text-align: center;
  background-color: #2e2d2d;
  color: white;
  padding: 10px;
  margin-top: 99px;
}