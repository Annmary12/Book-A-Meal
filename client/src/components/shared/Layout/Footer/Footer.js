import React from 'react';
import styles from './style.scss';

const Footer = () => <footer className={styles.footer}>&copy; 2018 Book-A-Meal Inc.</footer>;

export default Footer;
