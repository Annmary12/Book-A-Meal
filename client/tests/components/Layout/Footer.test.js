import React from 'react';
import Footer from '../../../src/components/Layout/Footer/Footer.js';

describe("Footer component", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<Footer />)

    expect(wrapper).toMatchSnapshot();
  })
})