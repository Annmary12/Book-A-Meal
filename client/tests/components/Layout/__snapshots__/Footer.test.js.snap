// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match snapshot 1`] = `
<footer
  className="footer"
>
  Â© 
  2018
   Book-A-Meal.
</footer>
`;
